pages/common/black.md

```diff
@@ -22,3 +22,7 @@
 - Auto-format a file or directory emitting exclusively error messages to stderr:
 
 `black --quiet {{path/to/file_or_directory}}`
+
+- Auto-format a file or directory without replacing single quotes with double quotes:
```

Comment:
Should we mention about [its use case from the documentation](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings)?
> adoption helper, avoid using this for new projects

Maybe like this?
> Auto-format a file or directory without replacing single quotes with double quotes (adoption helper, avoid using this for new projects)

Reply:
Hmmm, I'm not sure. We want to make sure the description isn't too long

Reply:
I'd include it. The description isn't too long and this sounds like important info.

Reply:
Hi @marchersimon, please review 0133152.